src/eins-hwinfo.c

```diff
@@ -130,30 +149,51 @@ record_disk_space (gpointer data)
 {
   g_autoptr(GFile) root = g_file_new_for_path ("/");
   g_autoptr(GFile) extra = g_file_new_for_path ("/var/endless-extra");
-  GVariant *root_payload = get_disk_space_for_partition (root);
-  GVariant *extra_payload = is_mounted (extra)
-    ? get_disk_space_for_partition (extra)
-    : g_variant_new ("(ttt)", 0, 0, 0);
-  GVariant *payload = g_variant_new ("(@(ttt)@(ttt))", root_payload, extra_payload);
+  g_autoptr(GError) error = NULL;
+  GVariant *root_payload =
+    eins_hwinfo_get_disk_space_for_partition (root, &error);
+
+  if (error != NULL)
+    {
+      g_warning ("Couldn't get disk space for /: %s", error->message);
+      g_clear_error (&error);
+    }
```

Comment:
Are you intentionally falling through here so as to return (0, 0, 0) in the error case?

Reply:
I am, but I could be convinced to just not submit anything in the error case.

Reply:
If (0, 0, 0) is to be reported and semantically meaningful, it should be documented - but the inclusion of such error values within the number space means we can't meaningfully rely on any aggregate operations like minimum, median, mode or mean without filtering the values first. I favor not submitting this value, because if we wished to count the occurrence of errors, we can infer the "NULL" value by whether the event is reported and compare to the number of reporting systems.

Reply:
It was documented in the ((ttt)(ttt)) version of this event but I lost it in the split. Anyway, your argument is convincing.